homeassistant/components/shelly/strings.json

```
@@ -440,6 +440,41 @@
         "name": "Water temperature"
       }
     },
+    "switch": {
+      "charging": {
+        "name": "Charging"
+      },
+      "child_lock": {
+        "name": "Child lock"
+      },
+      "frost_protection": {
+        "name": "Frost protection"
```

We should point here to the existing key to reduce translation efforts:
https://github.com/home-assistant/core/blob/b690bdc0e6010c7d1e516c078f34065c123cffa2/homeassistant/components/shelly/strings.json#L163

```suggestion
        "name": "%key:component::shelly::entity::climate::thermostat::state_attributes::preset_mode::state::frost_protection%]"
```